spec/registry.md

```diff
@@ -66,8 +66,33 @@ the corresponding input and options rules.
 If multiple `<override>` elements would match the current locale,
 only the first one is used.
 
-Matching-function signatures additionally include one or more `<match>` elements
-to define the keys against which they can match when used as selectors.
+### Variant Key Matches
+
+Matching-function signatures can include `<matches>` and `<when>` elements
+defining the variant keys matched by the selector.
+
+Each `<matches>` MAY contain either one or more `<match>` elements, or an `href` attribute.
+If an `href` attribute is set, its URL value MUST resolve to an XML document
+with a root `<matches>` element with no `href` attribute,
```

Comment:
Because we've no reason to, and this way we can rely on the external matches XML to resolve all of its dependencies completely.

Reply:
So there are no cases in which part of a `<matches>` tree in the external XML refers to (say) CLDR data? Once you've implemented resolving an external file, it's just a question of recursion, no?

Reply:
> So there are no cases in which part of a `<matches>` tree in the external XML refers to (say) CLDR data?

Correct. A `<matches>` can only contain `<match>` elements, which may only have `locales` and `values` attributes, and are otherwise empty. So there's no space for recursion.